product-service/handler.ts

```diff
@@ -1,5 +1,6 @@
 import 'source-map-support/register';
 import {getProductsList} from './handlers/getProductsList'
 import {getProductsById} from './handlers/getProductsById'
+import {createProduct} from './handlers/createProduct'
```

Comment:
you can export them this way:
`export * from  './handlers/createProduct'`
it's a bit shorter